test/__tests__/data/singular_router_input.json

```diff
@@ -340,5 +340,307 @@
+          "subscription_type": ""
+        },
+        "library": {
+          "name": "rudder-ios-library",
+          "version": "1.8.0"
+        },
+        "network": {
+          "wifi": true,
+          "carrier": "etisalat",
+          "cellular": false
+        },
+        "timezone": "Asia/Dubai",
+        "sessionId": 1674138119
+      },
+      "rudderId": "b7bbf406-aa7f-4d64-a388-925675e78f36",
+      "messageId": "b9e9534b-b3b7-4bdb-8d59-062e1fbf4e2b",
+      "timestamp": "2023-01-19T14:25:07.532Z",
+      "receivedAt": "2023-01-19T14:25:25.465Z",
+      "request_ip": "87.201.98.151",
+      "anonymousId": "cc06ff5e-838b-4e33-9d90-4ab5a6068bec",
+      "integrations": {
+        "All": true
+      },
+      "originalTimestamp": "2023-01-19T14:25:06.301Z"
+    },
+    "destination": {
+      "secretConfig": {},
+      "Config": {
+        "apiKey": "apiKey",
+        "apiSecret": "apiSecreth9h3984bnsdfkh98kjiu4h",
+        "sessionEventList": [
+          {
+            "sessionEventName": "test_event"
+          }
+        ],
+        "blacklistedEvents": [
+          {
+            "eventName": ""
+          }
+        ],
+        "whitelistedEvents": [
+          {
+            "eventName": ""
+          }
+        ],
+        "eventFilteringOption": "whitelistedEvents",
+        "eventDelivery": false,
+        "eventDeliveryTS": 1663919255086
+      },
+      "liveEventsConfig": {
+        "eventDelivery": false,
+        "eventDeliveryTS": 1663919255086
+      },
+      "ID": "2EhlBrBduwN2NX4pJ7nrVDHAz6C",
+      "workspaceId": "1sUXvPs0hYgjBxSfjG4gqnRFNoP",
+      "DestinationDefinition": {
+        "config": {
+          "destConfig": {
+            "ios": [
+              "useNativeSDK"
+            ],
+            "android": [
+              "useNativeSDK"
+            ],
+            "cordova": [
+              "useNativeSDK"
+            ],
+            "reactnative": [
+              "useNativeSDK"
+            ],
+            "defaultConfig": [
+              "apiKey",
+              "apiSecret",
+              "sessionEventList",
+              "blacklistedEvents",
+              "whitelistedEvents",
+              "eventFilteringOption"
+            ]
+          },
+          "excludeKeys": [],
+          "includeKeys": [
+            "apiKey",
+            "apiSecret",
+            "sessionEventList",
+            "blacklistedEvents",
+            "whitelistedEvents",
+            "eventFilteringOption"
+          ],
+          "transformAt": "processor",
+          "transformAtV1": "processor",
+          "supportedSourceTypes": [
+            "android",
+            "ios",
+            "flutter",
+            "reactnative",
+            "cordova",
+            "web",
+            "amp",
+            "cloud",
+            "warehouse"
+          ],
+          "saveDestinationResponse": true
+        },
+        "configSchema": {},
+        "responseRules": {},
+        "options": null,
+        "uiConfig": [
```

Comment:
Remove `uiConfig` section in both input & output --> this is redundant info in the test cases

Reply:
Done